### ISSUE TITLE
The referenced component could not be found

### ISSUE THREAD

**Issue by Huibuh2010** [qH0sT/AndroSpy#84]
Hey Folks,

i try to compile the Project but i cant build camera without a lot of errors. Can anybody help me plz? thanks

<img src="https://gfxf.net/images/2021/04/26/Unbenannt.jpg" alt="Unbenannt.jpg" border="0" />

**Comment by Huibuh2010** [comment]
Okay, it works now but only with Android 9. i tried install on android 10 device "this app is compiled for older version". How can i fix it? i tried project compile to android 10 framework but i didnt work.

**Comment by qH0sT** [comment]
> Okay, it works now but only with Android 9. i tried install on android 10 device "this app is compiled for older version". How can i fix it? i tried project compile to android 10 framework but i didnt work.

Do you use it on Emulator?

**Comment by Huibuh2010** [comment]
> Do you use it on Emulator?

Nope, Galaxy A40. i tried Android 9 on emulator and it works with android 9 but not on my telephone with android 10

**Comment by qH0sT** [comment]
> > Do you use it on Emulator?
> 
> Nope, Galaxy A40. i tried Android 9 on emulator and it works with android 9 but not on my telephone with android 10

write me on telegram: @v_for_vandet

**Comment by AngelBlaizeC** [comment]
Please post what is hte fix for this. I also encountered this error on android 10 and 11.

**Comment by Huibuh2010** [comment]
> 
> 
> Please post what is hte fix for this. I also encountered this error on android 10 and 11.

i dont have any fix... tried a lot but i doesnt work... i give up :D

**Comment by qH0sT** [comment]
@AngelBlaizeC 
@Huibuh2010 
Trojans or malwares or else set the TargetSDKInt value to 22 to obtain permissions automatically. For this reason this error occurs, try it with other RATs and you will see the same error. Otherwise, if we increase the value to 22, we will not be able to get permissions automatically.

**Comment by qH0sT** [comment]
Let me tell you a solution;
In Visual Studio, increase the TargetSDKInt (Target Android Version) of the project (26, 27, 28 ... or higher). Then rebuild the APK, then download and install it on the phone, but do not open it after it is installed; Go to the phone's Settings menu and go to the Installed Apps section, then find your APK, then activate all the off buttons from the Permissions section. Finally, open your APK by clicking on the main menu. This seems like a good solution but we must try :)

![Target](https://user-images.githubusercontent.com/45147475/116534559-57473f00-a8eb-11eb-81a2-7157e5384b93.png)
